.devcontainer/Dockerfile

```
@@ -0,0 +1,21 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.194.0/containers/python-3/.devcontainer/base.Dockerfile
+
+# [Choice] Python version: 3, 3.9, 3.8, 3.7, 3.6
+ARG VARIANT="3.9"
```

```suggestion
ARG PYTHON_VERSION="3.9"
```
to be consistent with `NODE_VERSION` later

---

It's based on the default Devcontainer [Dockerfile](https://github.com/microsoft/vscode-dev-containers/blob/main/containers/python-3/.devcontainer/Dockerfile). I personally would prefer to keep it consistent as the container obviously uses Python, but we could change that.